docs/source/user_guide/plots.rst

```diff
@@ -0,0 +1,969 @@
+visual dataset analysis pipeline.
+
+The builtin plots provided by FiftyOne are chosen to help you analyze and
+improve the quality of your datasets and models, with minimal customization
+required on your part to get started. At the same time, data/model
+interpretability is not a narrowly-defined space that can be fully automated.
+That's why FiftyOne's plotting framework is highly customizable and extensible,
+all by writing pure Python (no JavaScript knowledge required).
+
+.. note::
+
+    Check out the :ref:`tutorials page <tutorials>` for in-depth walkthroughs
+    that apply the available interactive plotting methods to perform
+    evaluation, identify model failure modes, recommend new samples for
+    annotation, and more!
+
+.. _overview:
+
+Overview
+________
+
+All |Session| instances provide a
+:meth:`plots attribute <fiftyone.core.session.Session.plots>` attribute that
+you can use to attach |ResponsivePlot| instances to the FiftyOne App.
+
+When |ResponsivePlot| instances are attached to a |Session|, they are
+automatically updated whenever
+:meth:`session.view <fiftyone.core.session.Session.view>` changes for any
+reason, whether you modify your view in the App, or programmatically change it
+by setting :meth:`session.view <fiftyone.core.session.Session.view>`, or if
+multiple plots are connected and another plot triggers a |Session| update!
+
+.. note::
+
+    Interactive plots are currently only supported in Jupyter notebooks. In the
+    meantime, you can still use FiftyOne's plotting features in other
+    environments, but you must manually call
+    :meth:`plot.show() <fiftyone.core.plots.base.Plot.show>` to update the
+    state of a plot to match the state of a connected |Session|, and any
+    callbacks that would normally be triggered in response to interacting with
+    a plot will not be triggered.
+
+    See :ref:`this section <working-in-notebooks>` for more information.
+
+Thw two main classes of |ResponsivePlot| are explained next.
+
+Interactive plots
+-----------------
+
+|InteractivePlot| is a class of plots that are bidirectionally linked to a
+|Session| via the IDs of either samples or individual labels in the dataset.
+When the user performs a selection in the plot, the
+:meth:`session.view <fiftyone.core.session.Session.view>` is automatically
+updated to select the corresponding samples/labels, and, conversely, when
+:meth:`session.view <fiftyone.core.session.Session.view>` changes, the contents
+of the current view is automatically selected in the plot.
+
+Examples of |InteractivePlot| types include
+:ref:`scatterplots <embeddings-plots>`,
+:ref:`location scatterplots <geolocation-plots>`, and
+:ref:`interactive heatmaps <confusion-matrices>`.
+
+.. image:: ../images/plots/mnist-interactive1.gif
+   :alt: interactive-plots
+   :align: center
+
+View plots
+----------
+
+|ViewPlot| is a class of plots whose state is automatically updated whenever
```

Comment:
Specifically state that they are not bidirectional. Also mention examples of types of `ViewPlots` here like for interactive plots.